eZ/Publish/Core/Repository/ContentService.php

```diff
@@ -1187,7 +1187,16 @@ public function updateContent(APIVersionInfo $versionInfo, APIContentUpdateStruc
         }
 
         $mainLanguageCode = $content->contentInfo->mainLanguageCode;
-        $languageCodes = $this->getLanguageCodesForUpdate($contentUpdateStruct, $content);
+        if ($contentUpdateStruct->initialLanguageCode === null) {
+            $contentUpdateStruct->initialLanguageCode = $mainLanguageCode;
+        }
+
+        $allLanguageCodes = $this->getLanguageCodesForUpdate($contentUpdateStruct, $content);
+        foreach ($allLanguageCodes as $languageCode) {
+            $this->persistenceHandler->contentLanguageHandler()->loadByLanguageCode($languageCode);
```

Comment:
Hmm, AFAICS this existed before, so maybe not an issue, but why is it needed if we don't use result?

Reply:
IMHO intention of this call in the original code was to check if a language exists. Ref:  https://github.com/ezsystems/ezpublish-kernel/blob/master/eZ/Publish/Core/Repository/ContentService.php#L1314

Reply:
Hmm, yep, that makes sense. Still weird to see it here. But let's keep it.